README.md

```diff
@@ -55,7 +56,7 @@ packages.
 
  * Save and exit.
 
- * Run `make`
+ * Run `make` (for a real benchmarking setup do *not* use `-j`)
```

Comment:
Yeah, OK. MIght be a while, currently trying to get grub out of the serial line on bencher8.